DOCS-11844 Update dataweave-formats-xml.adoc (#2022)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/dataweave-formats-xml.adoc

| `externalEntities` | `Boolean` | `false` | Indicates whether external entities
should be processed or not. By default this is disabled to avoid XXE attacks.
Valid values are `true` or `false`.

> | `indexedReader` | `Boolean` | `true` | DataWeave uses the indexed reader by default when reaching the threshold. Supports US-ASCII, UTF-8 and ISO-8859-1 encodings only. For other encodings, DataWeave uses the in-memory reader. Valid values are `true` or `false`. 

Reviewer: ```suggestion
| `indexedReader` | `Boolean` | `true` | DataWeave uses the indexed reader by default when reaching the threshold. Supports US-ASCII, UTF-8 and ISO-8859-1 encodings only. For other encodings and when set to `false`, DataWeave uses the in-memory reader. Valid values are `true` or `false`. 
```

> | `indexedReader` | `Boolean` | `true` | Indicates whether to use the indexed XML reader when the threshold is reached. Valid values are `true` or `false`. It only supports the encodings US-ASCII, UTF-8 and ISO-8859-1. For other encodings the in-memory reader will be used.

Reviewer: ```suggestion
| `indexedReader` | `Boolean` | `true` | DataWeave uses the indexed reader by default when reaching the threshold. Supports US-ASCII, UTF-8 and ISO-8859-1 encodings only. For other encodings, DataWeave uses the in-memory reader. Valid values are `true` or `false`. 
```
